style.css.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: blueviolet;
+}
```

Comment:
- Once you make all the chnages I suggested you do, I believe you will see how `blueviolet` can be on your `h1` :smile: